### PR TITLE
🔧 Attempt to fix the 'Rendering Device' setting

### PR DESCRIPTION
**Pull request by ulteq**
Fixes: https://github.com/RigsOfRods/rigs-of-rods/issues/1960

### CODE REVIEW

**Comment by AnotherFoxGuy** [comment]
Still launches on the same monitor

**Comment by AnotherFoxGuy** [comment]
`miscParams["monitorIndex"]` is always 0

**Comment by ulteq** [comment]
And what does `rd.possibleValues` contain?

File: source/main/gfx/OgreSubsystem.cpp [hunk 1]
```diff
@@ -92,6 +92,10 @@ bool OgreSubsystem::Configure()
     miscParams["vsync"] = ropts["VSync"].currentValue;
     miscParams["gamma"] = ropts["sRGB Gamma Conversion"].currentValue;
 #if OGRE_PLATFORM == OGRE_PLATFORM_WIN32
+    const auto rd = ropts["Render Device"];
```
Comment (AnotherFoxGuy) [inline source/main/gfx/OgreSubsystem.cpp]: It is called "Rendering Device"
```suggestion
    const auto rd = ropts["Rendering Device"];
```

**Pull request merged**